### DISCUSSION TITLE
LinkedInなど実名がわかるSNSへのリンクは非公開を設定可能にして欲しい

### DISCUSSION THREAD

**Discussion by lamrongol** [lapras-inc/lapras-user-community#17] [Ideas]
GitHubやZenn等と違い、LinkedIn等は実名登録せざるを得ないので、そうしたSNSへのリンクはPortfolioのWeb公開をオンにしていても非公開に設定できるようにして頂きたいです。

**Comment by UserSuccessTeamLAPRAS** [comment]
LAPRASサポートです。
ご意見ありがとうございます！改善のご要望として開発チームに共有させていただきました。

今後ともLAPRASをよろしくお願いします。
